Formula/planck.rb

```diff
@@ -21,8 +21,32 @@ class Planck < Formula
   depends_on "icu4c"
   depends_on "libzip"
 
+  uses_from_macos "curl"
+
+  on_linux do
+    depends_on "vim" => :build # for xxd
+    depends_on "gcc"
+    depends_on "glib"
+    depends_on "libffi"
+    depends_on "pcre"
+    depends_on "webkitgtk"
+  end
+
+  fails_with gcc: "5"
+
+  # Apply upstream commit to fix issue with GNU sed.  Remove with next release.
+  patch do
+    url "https://github.com/planck-repl/planck/commit/f1f21bf9eb4cfca6312ff0117f75d3b38164b43d.patch?full_index=1"
+    sha256 "787ddf6fb1cfea1d70fa18a6f7b292579ea1ffbf1f437f1f775e3330d2b8d36c"
+  end
+
   def install
     ENV["JAVA_HOME"] = Formula["openjdk"].opt_prefix
+    ENV.prepend_path "PATH", Formula["openjdk"].opt_bin unless OS.mac?
+
+    # Make sure CMake detects javascriptcoregtk-4.1 on Linux.  Remove when this file is updated to recognize 4.1.
+    inreplace "planck-c/CMakeLists.txt", "javascriptcoregtk-4.0", "javascriptcoregtk-4.1"
```

Comment:
Where does our `4.1` come from?

Reply:
That comes from the name of the `pkg-config` file in `webkitgtk`.  It is the API version of `javascriptcore`.  Should this maybe be autogenerated?  We just did this for libetonyek so I can reuse the same approach.